fontique/src/backend/coretext.rs

```diff
@@ -37,7 +41,17 @@ pub struct SystemFonts {
 
 impl SystemFonts {
     pub fn new() -> Self {
-        let scanned = scan::ScannedCollection::from_paths(Some("/System/Library/Fonts"), 8);
+        let paths = unsafe {
+            NSSearchPathForDirectoriesInDomains(
+                NSSearchPathDirectory::NSLibraryDirectory,
+                NSSearchPathDomainMask::NSAllDomainsMask,
+                Bool::YES,
+            )
+            .as_ref()
```

Comment:
`as_ref` in this context is fine as long as you do not return it from this function (the array is placed in the autorelease pool, so that would be unsound). Another alternative would be `Retained::retain_autoreleased(NSSearchPathForDirectoriesInDomains(...).as_ptr()).unwrap()`.

At some point this will be solved in `objc2`.

Reply:
We're not returning it, so I think just using `as_ref` is fine for now (and certainly shorter than `Retained::retain_autoreleased(...)`)